src/main/java/com/jcabi/s3/Bucket.java

```diff
@@ -108,7 +108,29 @@ void remove(@NotNull(message = "S3 key can't be NULL") String key)
         String pfx) throws IOException;
 
     /**
-     * Prefixed.
+     * Creates bucket with specified origin bucket and prefix.
+     *
+     *
+     * <p>Basically this class is involved to cut off ocket keys of underlying
```

Comment:
I think it will be better with such change: *... involved to cut off ...* -> *... is used to cut off ...*